views/home.erb

```diff
@@ -0,0 +1,10 @@
+<div style="text-align: center">
+  <h1>Chitter:</h1>
+
+  <form action="/" method="post">
+    <input name="message" type="text" placeholder="Write a new Peep..."/>
+    <button value="Submit" type="submit">Peep!</button>
+  </form>
+
+  <p><%= @all_peeps.join('/n') %></p>
```

Comment:
This wouldn't work - ('/n') would just show up as a string. Maybe "/n" (full quotation marks) would work?

Reply:
What's the HTML equivalent of a line break? That's the thing to research.